Client/mods/deathmatch/logic/CClientColPolygon.cpp

```diff
@@ -28,6 +28,11 @@ bool CClientColPolygon::DoHitDetection(const CVector& vecNowPosition, float fRad
     if (!IsInBounds(vecNowPosition))
         return false;
 
+    float z = vecNowPosition.fZ;
+
+    if (z > m_fCeil || z < m_fFloor)
```

Comment:
Unnecessary variable.
```suggestion
    if (vecNowPosition.fZ > m_fCeil || vecNowPosition.fZ < m_fFloor)
```